flygym/fly.py

```diff
@@ -939,7 +962,7 @@ def _add_adhesion_actuators(self, gain):
 
     def set_pose(self, pose: state.KinematicPose, physics: mjcf.Physics):
         for i in range(len(self.actuated_joints)):
-            curr_joint = self.actuators[i].joint.name
+            curr_joint = self.actuators[i].joint
```

Comment:
Why is `.name` no longer needed?

Reply:
For an unknown reason self.actuators[I].joints yields a joint object if obtained through the xml but only a string if the actuator is added in the code.